visionipc/visionipc_client.cc

```diff
@@ -9,7 +9,12 @@
 #include "cereal/logger/logger.h"
 
 static int connect_to_vipc_server(const std::string &name, bool blocking) {
+  char* prefix = std::getenv("OPENPILOT_PREFIX");
   std::string path = "/tmp/visionipc_" + name;
+  if (prefix) {
+    path = path + "_" + prefix;
```

Comment:
should be prefix + path, with a default prefix of /tmp/

Reply:
or maybe an intermediate directory of /tmp/ + prefix + '/visionipc_' + name